tools/lan95xx-set-mac

```diff
@@ -2,33 +2,15 @@
 #
 # SPDX-License-Identifier: GPL-2.0
 #
-# Copyright: 2022 KUNBUS GmbH
+# Copyright: 2022-2023 KUNBUS GmbH
 #
-set -e
 
 iface="$1"
 mac="$2"
-mac="${mac,,}"		# downcase
-mac="${mac//-/}"	# remove dashes
-mac="${mac//:/}"	# remove colons
 
-if [ "$#" != 2 ] ||
-   ! [[ "$mac" =~ ^([0-9a-f]{2})([0-9a-f]{2})([0-9a-f]{2})([0-9a-f]{2})([0-9a-f]{2})([0-9a-f]{2})$ ]] ; then
-	echo 1>&2 "Usage: $(basename "$0") <eth interface> <mac address>"
-	exit 1
+if [ "$#" != 2 ]; then
+    echo 1>&2 "Usage: $(basename "$0") <eth interface> <mac address>"
+    exit 1
 fi
 
-# write golden image to EEPROM
-LAN95XX_BIN="/usr/share/revpi/lan9514.bin"
-if [ -f "${LAN95XX_BIN}" ]; then
-	ethtool -E "${iface}" magic 0x9500 offset 0 length 256 < "${LAN95XX_BIN}"
-else
-	echo 1>&2 "The golden image doesn't exist"
-	exit 1
-fi
-
-# write MAC Address (The offset of MAC are the same in LAN950X, 9512, 9513, 9514)
```

Comment:
The comment in parentheses should probably be retained somewhere, otherwise the information is lost.

Reply:
This is marked resolved, but I'm not seeing the information retained anywhere?

Reply:
Never mind, I found it.